src/io/flutter/run/bazelTest/BazelTestFields.java

```diff
@@ -138,7 +167,16 @@ BazelTestFields copyTemplateToNonTemplate(@NotNull final Project project) {
    * @throws RuntimeConfigurationError for an error that that the user must correct before running.
    */
   void checkRunnable(@NotNull final Project project) throws RuntimeConfigurationError {
-    BazelFields.checkRunnable(project, getEntryFile(), getLaunchingScript(), getBazelTarget());
+    // The UI only shows one error message at a time.
+    // The order we do the checks here determines priority.
+
+    final DartSdk sdk = DartPlugin.getDartSdk(project);
+    if (sdk == null) {
```

Comment:
Do you need a call to sdk.getVersion().flutterTestSupportsFiltering(); here as well, like above?

Reply:
No, and for two reasons:

- bazel test filtering is managed separately from the Flutter SDK AFAIK.  Test filtering support was added at a later change in g3 than when it was added to the Flutter SDK.
- when the test runner is not available, all test configurations will use the legacy behavior, which doesn't attempt to filter targets.